patches/components-flags_ui-resources-flags.html.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/components/flags_ui/resources/flags.html b/components/flags_ui/resources/flags.html
+index 46df47e9f54938051c0404f8ebb5d0ea953706e8..9292462a9bda35ad62b85c96601a4fb09ff978cd 100644
+--- a/components/flags_ui/resources/flags.html
++++ b/components/flags_ui/resources/flags.html
+@@ -225,6 +225,7 @@
+   </div>
+ </div>
+ <script src="flags.js"></script>
++<script src="chrome://brave-resources/page_specific/flags/flags_overrides.js"></script>
```

Comment:
I think we can probably get rid of a substantial number of patches by overriding `BraveContentClient::GetDataResourceBytes` to return the original resource `flags.js` with `flags_overrides.js` appended to it.

Reply:
cc @petemill